src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1071,8 +1076,11 @@ def auto_shutdown_vm(cmd, resource_group_name, vm_name, off=None, email=None, we
     return client.global_schedules.create_or_update(resource_group_name, name, schedule)
 
 
-def get_instance_view(cmd, resource_group_name, vm_name):
-    return get_vm(cmd, resource_group_name, vm_name, 'instanceView')
+def get_instance_view(cmd, resource_group_name, vm_name, include_user_data=False):
```

Comment:
Why `include_user_data` is added here but ignored in `_params.py`?

Reply:
This is because `get_instance_view()` is not only a custom method called by `vm get-instance-view`, `vm wait` and `vmss wait` commands, but also called by method `get_vm_details()`.
When `get_instance_view()` is used in `get_vm_details()` command, it adds a parameter `include_user_data`, which needs to be passed into `get_instance_view()` method to modify the logic of expand.
Because this parameter is not required for `vm get-instance-view`, `vm wait` and `vmss wait` commands, so it is ignored in these three commands.